index.d.ts

```diff
@@ -67,10 +67,32 @@ const theme: {
 interface StyleProps
   extends FlexboxProps,
     SpaceProps,
-    BorderProps,
-    ColorProps,
+    Omit<
+      BorderProps,
+      | "borderColor"
+      | "borderTopColor"
+      | "borderBottomColor"
+      | "borderLeftColor"
+      | "borderRightColor"
+    >,
+    Omit<ColorProps, "backgroundColor" | "bg" | "color">,
```

Comment:
@kamolesh97 I guess without omitting also it will work, As we are overriding at line  81. Please check once